### PR TITLE
show outputs of HLT unit tests

### PR DESCRIPTION
**Pull request by missirol**
#### PR description:

This PR applies small technical tweaks to some of the unit tests used in HLT packages.

  - The main change is to show the outputs of the tests in stdout/err, as opposed to redirecting them to a log file.
  - Other changes are just to make the structure+naming of the tests more uniform across subpackages.

#### PR validation:

Unit tests pass.

#### If this PR is a backport, please specify the original PR and why you need to backport that PR. If this PR will be backported, please specify to which release cycle the backport is meant for:

N/A

### CODE REVIEW

**Comment by missirol** [comment]
please test

**Comment by missirol** [comment]
+hlt

Unit tests pass, and their logs are as expected.

File: HLTrigger/JSONMonitoring/test/testHLTriggerJSONMonitoring_cfg.py [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import FWCore.ParameterSet.Config as cms
 
 process = cms.Process("TEST")
+
 process.options.numberOfThreads = 1
+process.options.numberOfStreams = 0
```
Comment (perrotta) [inline HLTrigger/JSONMonitoring/test/testHLTriggerJSONMonitoring_cfg.py:6]: Do you really want to define and then overwrite `numberOfThreads`?

Comment (missirol) [reply HLTrigger/JSONMonitoring/test/testHLTriggerJSONMonitoring_cfg.py:6]: Mh.. this is number of streams?

Comment (perrotta) [reply HLTrigger/JSONMonitoring/test/testHLTriggerJSONMonitoring_cfg.py:6]: > Mh.. this is number of streams?

You are right: shame on me!

**Comment by perrotta** [comment]
+1

**Pull request merged**